### ISSUE TITLE
Script module (fake module btw, whats up with that) bleeds stdout into stderr

### ISSUE THREAD

**Issue by Jmainguy** [ansible/ansible#12874]
From bcoca in last ticket
https://github.com/ansible/ansible-modules-core/issues/2276

```
the script module does not call ssh, ansible does, in 1.9x ssh always gets called -v so ansible can figure return a meaningful error if the connection fails, ssh always writes debug/verbose info to stderr.

As such this is not a bug in the script module, but with the ssh connection plugin bleeding the stderr into the module results.
```

In Ansible 1.9.3 the script module throws a ton of ugly ssh connection info into stderr. Where as there should be no ugly ssh connection info if it worked, and should through the script result into stdout (if it worked).

**Comment by Jmainguy** [comment]
Example run in 1.9.3

```
cee-insights01.web.qa.ext.datacenter.example.com | success >> {
    "changed": true,
    "rc": 0,
    "stderr": "OpenSSH_6.9p1, OpenSSL 1.0.1k-fips 8 Jan 2015\r\ndebug1: Reading configuration data /home/rdu/jmainguy/.ssh/config\r\ndebug1: /home/rdu/jmainguy/.ssh/config line 21: Applying options for *.datacenter.example.com\r\ndebug1: Reading configuration data /etc/ssh/ssh_config\r\ndebug1: /etc/ssh/ssh_config line 56: Applying options for *\r\ndebug1: auto-mux: Trying existing master\r\ndebug1: mux_client_request_session: master session id: 2\r\nShared connection to cee-insights01.web.qa.ext.datacenter.example.com closed.\r\n",
    "stdout": "LAAAAME\r\n"
}
```

**Comment by Jmainguy** [comment]
Closing this out, its fixed in 2.0

```
[root@phy01 ansible]# ansible -i hosts all -m script -a "neat.sh"
phy02.standouthost.com | SUCCESS => {
    "changed": true, 
    "rc": 0, 
    "stderr": "", 
    "stdout": "abadger1999\r\n", 
    "stdout_lines": [
        "abadger1999"
    ]
}
```
